### DISCUSSION TITLE
Nautilus integration

### DISCUSSION THREAD

**Discussion by aayushrautela** [andyholmes/valent#621] [Feature Request]
As gsconnect supprts sending files directly from nautilus, it would be nice to have the same functionality with valent's helper extension.

**Comment by andyholmes** [comment]
I don't plan on shipping a Nautilus extension this time, but if you wanted to make one it would be almost a copy-paste of the GSConnect one.

It just doesn't really add much benefit IMO, since you can just open files with Valent and it will offer to send them.

**Reply by aayushrautela** [reply]
Thanks for the clarification. However sending with share menu is multi step process,with extra step of searching valent in share menu and then selecting device.

**Reply by andyholmes** [reply]
Yes, that's true. The existing script should be pretty easy to adapt fortunately :slightly_smiling_face:
